homeassistant/components/drop_connect/strings.json

```diff
@@ -31,6 +31,7 @@
       "pending_notification": { "name": "Notification unread" },
       "reserve_in_use": { "name": "Reserve capacity in use" },
       "salt": { "name": "Salt low" },
+      "alert_sensor": { "name": "Sensor" },
```

Comment:
The generic naming of the sensor input on this device is intentional based on the many ways that it can be used on a DROP system.  For example, it can be used with a float switch to detect rising a water level in a sump pit, notify users that pump equipment has entered a fault state, detect water intrusion, or shut off the water due to a dry contact closure on an alarm panel.  Because of this, the sensor input really doesn't have a better name that wouldn't be incorrect in some cases.  We spent quite a bit of time considering alternatives when the marketing plan for the device was formed and decided it was better to be vague instead of misleading.

That being said, we're open to suggestions but haven't come up with anything that is an improvement.